public/app/features/explore/state/reducers.test.ts

```diff
@@ -174,6 +175,50 @@ describe('Explore item reducer', () => {
     });
   });
 
+  describe('changing refresh intervals', () => {
+    it("should result in 'streaming' state, when live-tailing is active", () => {
+      const initalState = makeExploreItemState();
+      const expectedState = {
+        ...initalState,
```

Comment:
I think this could potentially lead to false positive tests if the item state would be a deep object structure, might not be that now though. This suggestion should be safer. Could you change to this solution in the rest of this file?
```suggestion
        ...makeExploreItemState(),
```